### PR TITLE
Add OceanBase

### PR DESCRIPTION
**Pull request by eddumelendez**

### CODE REVIEW

**Comment by eddumelendez** [comment]
@whhe Can you please take a look at this? Once the new Testcontainers for Java version is release, the module would be available in the [Module Catalog](https://testcontainers.com/modules/).

**Review by whhe — APPROVED** [review]
Thanks for your great work, LGTM.

File: modules/oceanbase.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - id: java
+    url: https://java.testcontainers.org/modules/databases/oceanbase/
+    isThirdParty: true
+    example: |
+      ```java
+      OceanbaseCEContainer oceanbase = new OceanbaseCEContainer("oceanbase/oceanbase-ce:4.2.2");
+      oceanbase.start();
+      ```
+description: |
+  OceanBase is an open source distributed SQL database.
```
Comment (whhe) [inline modules/oceanbase.md:15]: Here is a more detailed copy from the website.

> OceanBase is an open source unlimited scalable distributed database for data-intensive transactional and real-time operational analytics workloads, with ultra-fast performance that has once achieved world records in the TPC-C benchmark test.

Comment (eddumelendez) [inline modules/oceanbase.md:15]:
```suggestion
  OceanBase is an open source unlimited scalable distributed database for data-intensive transactional and real-time operational analytics workloads, with ultra-fast performance that has once achieved world records in the TPC-C benchmark test.
```

**Pull request merged**